spec/features/orders/creation_spec.rb

```diff
@@ -132,36 +132,6 @@
       expect(page).to have_content("New York")
       expect(page).to have_content(10001)
     end
-    it "Non pending order doesnt display" do
```

Comment:
Deleted this with Danny's permission since it doesn't say in the user story that it is required, and another user story indicated that all orders should display (not just pending)